portfolio/src/main/webapp/script.js

```diff
@@ -40,6 +40,23 @@ function addFavoriteThing() {
     factContainer.innerText = 'That is all the info you get for now...';
   }
 }
+
+function getUserInfo() {
+  fetch('login-check').then((response) => response.json()).then((loginInfo) => {
+    if (loginInfo.isLoggedIn) {
+      console.log(loginInfo.logoutLink);
+      document.getElementById("comment-form").style.display = "block";
+      document.getElementById("login-logout").href = loginInfo.logoutLink;
```

Comment:
this link would be same regardless of logout or login except for the inner text. do you want to move it outside and just call it a link

Reply:
Right, didn't think about that. I moved it outside.